diff -u -Naur opengrm-ngram-1.2.1~/configure.ac opengrm-ngram-1.2.1/configure.ac
--- opengrm-ngram-1.2.1~/configure.ac	2015-01-02 20:28:11.921727970 +0000
+++ opengrm-ngram-1.2.1/configure.ac	2015-01-02 20:28:37.911728307 +0000
@@ -6,6 +6,7 @@
 AC_PROG_CXX
 AC_DISABLE_STATIC
-AC_PROG_LIBTOOL
+AM_PROG_AR
+LT_INIT
 
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_SRCDIR([src/include/ngram/ngram-model.h])
